docs/paper/dev/api/command-api/arguments-and-literals.mdx

```diff
@@ -0,0 +1,189 @@
+    )
+
+    .then(Commands.literal("survival")
+        .executes(ctx -> /* Set player's gamemode to survival */)
+    )
+
+    .then(Commands.literal("spectator")
+        .executes(ctx -> /* Set player's gamemode to spectator */)
+    )
+
+    .then(Commands.literal("adventure")
+        .executes(ctx -> /* Set player's gamemode to adventure */)
+    );
+```
+
+You may have noticed the use of the `.executes(...)` method. This method defines the execution logic of a node. If the player sends `/gamemode`, without any arguments, it
+will not even run, as our root node does not have a executor defined. Contrary, if the player executes `/gamemode creative`, it will run the code in the first `executes` method.
+We are not going to go into detail of the stuff that you can do inside a `execute` method, as that is out of scope for this site. If you want to read up on those, you should
+check out the [Extensive Guide on Command Execution](./command-executors) after you finish reading this.
+
+## Arguments
+Arguments are slightly more complex. They also define a new branch in a tree, but they are not directly predictable. Each argument is created using `Commands.argument(String, ArgumentType<T>)`.
+That method returns a `RequiredArgumentBuilder`. The T type parameter declares the return type of the argument, which you can then use inside your `executes` method. That means that
+if you put in an `ArgumentType<Integer>`, you can retrieve the value of that argument as an integer, requiring no manual parsing! The only issue is, how do we create an `ArgumentType`
+object? There are a few build-in, primitive argument types that you should definitely be aware of:
+
+|               Name                | Return value  |  Possible Input   |                                    Description                                     |
+|-----------------------------------|---------------|-------------------|------------------------------------------------------------------------------------|
+| BoolArgumentType.bool()           | Boolean       | true/false        | Only allows a boolean value                                                        |
+| IntegerArgumentType.integer()     | Integer       | 253, -123, 0      | Any valid integer                                                                  |
+| LongArgumentType.longArg()        | Long          | 25418263123783    | Any valid long                                                                     |
+| FloatArgumentType.floatArg()      | Float         | 253.2, -25.0      | Any valid float                                                                    |
+| DoubleArgumentType.doubleArg()    | Double        | 4123.242, -1.1    | Any valid double                                                                   |
+| StringArgumentType.word()         | String        | no_spaces         | A single word. May only contain letters and underscores                            |
```

Comment:
The `StringArgumentType.word()` supports alphanumeric values, and these: `+`, `-`, `_`, `.`.

Reply:
Will update the description once I will be revising that file, thanks for clearing that up!